src/SOS/debugshim/debugshim.cpp

```diff
@@ -422,193 +422,196 @@ HRESULT CLRDebuggingImpl::GetCLRInfo(ICorDebugDataTarget* pDataTarget,
                                      __out_z __inout_ecount(dwDacNameCharCount) WCHAR* pDacName,
                                      DWORD  dwDacNameCharCount)
 {
-#ifndef FEATURE_PAL
```

Comment:
What about leaving this code ifndef FEATURE_PAL but moving the xplat code below out of the ifdef and in the "else" of the "if (m_isWindowsTarget)".

So on Linux, the Windows code is still ifdef'ed out but on Windows both the Windows/Linux code is present.

Reply:
Thaat's what I was thinking too, even without knowing why the build was failing.